contracts/prize-strategy/PrizeStrategy.sol

```diff
@@ -437,4 +440,5 @@ contract PrizeStrategy is PrizeStrategyStorage,
     bytes calldata operatorData
   ) external override {
   }
+  
 }
```

Comment:
I wonder if the "beforeTokenTransfer" function should check the RNG service status `(isRngRequested && !isRngCompleted)` and return a Boolean for success.  Then the PrizePool can revert on "depositTo" if the RNG is locked

Reply:
That's handled in another PR